tools/l10n/extract_pot.cmake

```diff
@@ -5,6 +5,8 @@ file(GLOB_RECURSE cpp_files "${base_dir}/source/*.cpp" RELATIVE "${base_dir}" LI
 
 set(ror_pot "${base_dir}/languages/ror.pot")
 
+file(WRITE ${ror_pot} "")
```

Comment:
this isn't "adding _L macro to all strings", it looks like some bugfix.
I'd like it to be in a separate commit